src/models/user.repository.ts

```diff
@@ -0,0 +1,90 @@
+import { User } from "./models";
+import { Item } from "./models";
+
+/**
+ * @desc Create a new user
+ * @param {User} user
+ * @returns {Promise<User | boolean>} Created user or false if user already exists
+ */
+export const createUser = async (user: User): Promise<User | boolean> => {
+  // Check if user already exists
+  const userExists = await User.findOne({
+    where: {
+      username: user.username,
+    },
+  });
+  if (userExists) {
+    return false;
+  }
+  // Create user
+  const newUser = await User.create(user);
+  return newUser;
+};
+
+/**
+ * @desc Check if user exists and return user
+ * @param {User} user object
+ * @returns {Promise<User | null>} Username or false if user does not exist or password is incorrect
+ */
+export const loginUser = async (user: User): Promise<User | null> => {
```

Comment:
```suggestion
export const userExists = async (user: User): Promise<User | null> => {
```

Should probably change this function name to `userExists` or something similar; we are not actually logging in a user in this function.